### PR TITLE
Fixed a typo in README.md

### PR DESCRIPTION
**Pull request by elseju**
In the context, "then" is more suitable than "than".

### CODE REVIEW

File: search.js [hunk 1]
```diff
@@ -1 +1,22 @@
 // workshop code goes in here :)
+var search = instantsearch({
+  appId: 'latency',
+  apiKey: '0e5d8dce55757ee9b0d8bafadd2ec1ed',
+  indexName: 'movies',
+  urlSync: {}
+});
+
+search.addWidget(
+  instantsearch.widgets.searchBox({
```
Comment (Haroenv) [inline search.js]: could you remove the solution from this commit?

**Review by Haroenv — CHANGES_REQUESTED** [review]
Great change, but you accidentally committed the solution here too. Can you change that?

**Comment by Haroenv** [comment]
You should be able to do `git revert -n ca1921a5f64031e331d1ab092e6c7379ad9de66b`

**Review by Haroenv — APPROVED** [review]

**Comment by Haroenv** [comment]
Thank you @ellieseoejung 🔥

**Comment by elseju** [comment]
Yay! My first big PR :D ❤️

**Comment by Haroenv** [comment]
looking forward to what else you can PR 😄 👌

**Pull request merged**